shared/js/background/chrome-events.es6.js

```diff
@@ -246,6 +279,10 @@ chrome.alarms.create('updateLists', { periodInMinutes: 30 })
 chrome.alarms.create('updateUninstallURL', { periodInMinutes: 10 })
 // remove expired HTTPS service entries
 chrome.alarms.create('clearExpiredHTTPSServiceCache', { periodInMinutes: 60 })
+// Update userAgent lists
+chrome.alarms.create('updateUserAgentData', { periodInMinutes: 12 * 60 })
+// Rotate the user agent spoofed
+chrome.alarms.create('rotateUserAgent', { periodInMinutes: 24 * 60 })
```

Comment:
Just double checking... It looks like this will rotate in two cases:
1. 24 hr alarm triggers
2. browser restarts and there's a new etag

Is that correct?

Reply:
That's right - new data or alarm trigger